app/components/app-toggle-card.component.scss

```diff
@@ -5,6 +5,8 @@
 }
 
 .app-toggle-card {
+    cursor: none;
```

Comment:
what's this change about?

Reply:
the toggle card in lawfirm app has a cursor when you hover over the entire vacation mode card but the card is not clickable. the only part of the toggle card that should have a cursor is the toggle.